internal/compiler/tests/syntax/focus/initial_focus_non_input.slint

```diff
@@ -41,7 +41,6 @@ export Y := FocusScope {
     key-pressed => {
         r0.focus();
         x.focus();
-//      ^error{focus\(\) can only be called on focusable elements}
```

Comment:
Should we not still reject this?

Reply:
I think this is actually slightly better. Given this:

```
clicked => { x.focus(); }
x := SomeElement {}
```

and

```
component SomeElement {
    forward-focus: bad;
    bad := Rectangle {}
}
```

The error is at the bad `forward-focus` declaration, not at `focus()` call site. And that's where it'll be reported. Reporting it also at the `x.focus()` site is not very actionable.

Reply:
it is better in this case, indeed.